docs/Serverless.md

```diff
@@ -0,0 +1,99 @@
+<h1 align="center">Serverless</h1>
+
+Run serverless applications and REST APIs using your existing Fastify application.
+
+> Fastify is not designed to run on the AWS Lambda platform.
+The Fastify framework is designed to make implementing a traditional HTTP/S server easy.
+The AWS Lambda platform services requests in a completely different manner than
+a traditional HTTP/S server and is therefore at odds with Fastify's whole design.
+Regardless, it is possible to use Fastify in this environment if you so choose.
+This document describes how to do it. However, remember that
+this is an unsupported usage of Fastify and you are on your own in terms of support.
+
+
+## Serverless environments
+
+The sample provided allows you to easily build serverless web applications/services
+and RESTful APIs using Fastify on top of AWS Lambda and Amazon API Gateway.
+
+
+Having a normal Fastify application like this:
+
+```js
+const fastify = require('fastify');
+
+const app = fastify({ serverFactory });
+
+app.get('/', (request, reply) => reply.send({ hello: 'world' }));
+
+app.listen(3000, () => console.log('server listening on 3000'));
+```
```

Comment:
I don't see the purpose of the sentence on line 20 and the above code block. You seem to capture this in the below example.

Reply:
✅